crates/re_string_interner/src/lib.rs

```diff
@@ -84,7 +84,7 @@ impl nohash_hasher::IsEnabled for InternedString {}
 
 impl std::cmp::PartialOrd for InternedString {
     fn partial_cmp(&self, other: &Self) -> Option<std::cmp::Ordering> {
```

Comment:
unrelated, but we should make this `#[inline]`

```suggestion
    #[inline]
    fn partial_cmp(&self, other: &Self) -> Option<std::cmp::Ordering> {
```